playout/libretime_playout/liquidsoap/templates/outputs.liq.j2

```diff
@@ -41,6 +41,44 @@
 {%- endif -%}
 {%- endmacro -%}
 
+
+{#-
+  Build an hls output the output configuration.
+#}
+{%- macro output_hls(output_id, output) -%}
+# hls:{{ output_id }}
+
+output_hls_{{ output_id }}_source = s
+
+streams = ref([])
+
+{% for stream in output.streams -%}
+
+    streamout = %ffmpeg(format="{{output.format}}" ,
```

Comment:
note that %ffmpeg (as well as %audio for liquidsoap>2.0) arguments can't be dynamically passed through a function we can cursomize for different lq versions (as i did for output.file.hls).....So we can't maintain hls for both lq 1.4 and 2.0

Reply:
For the futur, if you really need to handle the %functions, you may use jinja templating and check against the version during the startup of playout.